src/haxelib/client/Main.hx

```diff
@@ -188,6 +188,14 @@ class Main {
 	}
 
 
+	function checkUpdate() {
+		var latest:SemVer = try site.getLatestVersion("haxelib_client") catch (_:Dynamic) null;
```

Comment:
The library is going to be `haxelib` instead of `haxelib_client` as were discussed.